_includes/v23.1/cdc/privilege-model.md

```diff
@@ -11,9 +11,23 @@ You can [grant](grant.html#grant-privileges-on-specific-tables-in-a-database) a
 GRANT CHANGEFEED ON TABLE example_table TO user;
 ~~~
 
-This privilege model provides a more granular way to grant users the ability to create a changefeed on a table. A user granted the `CHANGEFEED` privilege can create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+When you grant a user the `CHANGEFEED` privilege, they can:
```

Comment:
"... privilege on a set of tables, they can:"

Reply:
Added!